config/examples/redhat_customer_portal.yml

```diff
@@ -1,19 +1,15 @@
 base: &default
-  enabled: true
-  client_id:
-  client_secret:
-  realm:
-  flow:
-  entitlements_notifications_enabled: true
+  enabled: <%= ENV.fetch('RH_CUSTOMER_PORTAL_ENABLED', false) %>
+  client_id: <%= ENV.fetch('RH_CUSTOMER_PORTAL_CLIENT_ID', '3scale') %>
+  client_secret: <%= ENV['RH_CUSTOMER_PORTAL_CLIENT_SECRET'] %>
+  realm: <%= ENV.fetch('RH_CUSTOMER_PORTAL_REALM', 'implicit') %>
```

Comment:
what does this `implicit` mean here? I see different values before.

Reply:
Oops, this is is actually a bug. Good catch!
It should be a URL pointing to a RH-SSO realm.